src/components/dev-hub/podcast-player/audio-player.js

```diff
@@ -0,0 +1,236 @@
+import React, { useCallback, useReducer, useRef } from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+import ReactPlayer from 'react-player/lazy';
+
+import PlayerControls, {
+    ControlsContainer,
+} from '~components/dev-hub/podcast-player/player-contols';
+import PlayerListenOn, {
+    LogosContainer,
+} from '~components/dev-hub/podcast-player/player-listen-on';
+import Slider, { SliderContainer } from '~components/dev-hub/slider';
+
+import { getTimeLabel } from '~utils/get-time-label';
+import { size, colorMap } from '~components/dev-hub/theme';
+
+import {
+    PLAYER_ACTIONS,
+    playerInitialState,
+    playerReducer,
+} from '~components/dev-hub/podcast-player/player-reducer';
+
+const APPLE_PODCASTS_URL =
+    'https://podcasts.apple.com/us/podcast/the-mongodb-podcast/id1500452446';
+const SPOTIFY_PODCASTS_URL =
+    'https://open.spotify.com/show/0ibUtrJG4JVgwfvB2MXMSb?si=k1oOQ8JcSr6WtfeSMQ-kFA&nd=1';
+const GOOGLE_PODCASTS_URL =
+    'https://podcasts.google.com/feed/aHR0cHM6Ly9tb25nb2RiLmxpYnN5bi5jb20vcnNz?sa=X&ved=2ahUKEwiE97rnkfDrAhU1n3IEHanUCRUQ9sEGegQIARAC';
+
+const mobileSliderStyles = css`
+    display: grid;
+    grid-template-areas:
+        'slider slider'
+        'current  total';
+    justify-content: space-between;
+
+    input {
+        grid-area: slider;
+    }
+    p {
+        &:first-of-type {
+            grid-area: current;
+            margin-left: ${size.tiny};
+        }
+        &:last-of-type {
+            grid-area: total;
+            margin-right: ${size.tiny};
+            text-align: end;
+        }
+    }
+`;
+
+const ContentContainer = styled('div')`
+    display: flex;
+    justify-content: center;
+`;
+
+const PlayerContainer = styled('div')`
+    align-items: flex-start;
+    background-color: ${colorMap.devBlack};
+    border-radius: ${size.xsmall};
+    border: ${colorMap.greyDarkTwo} 1px solid;
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+    padding: 30px;
+    width: 100%;
+`;
+
+const AudioContainer = styled('div')`
+    align-items: center;
+    display: flex;
+    width: 100%;
+
+    @media only screen and (max-width: 600px) {
+        flex-direction: column;
+
+        ${ControlsContainer} {
+            margin-bottom: ${size.small};
+            a:last-of-type {
+              margin-right: 0;
+            }
+        }
+    }
+`;
+
+const StyledSliderContainer = styled('div')`
+    margin-left: ${size.default};
+    width: 100%;
+
+    @media only screen and (max-width: 600px) {
```

Comment:
I know the Figma uses these values for the mobile/tablet breakpoints but let's defer to our consistent values in the `screenSize` object within the `theme`. 

```suggestion
    @media ${screenSize.upToMedium} {
```